flutter-mentorship-t4/mentorship_project
# feat (Cart) 🎇 finish cart UI

# Pull Request: Cart UI in Flutter App

## Jira Ticket: [DIVA-17](https://nz-dev.atlassian.net/jira/software/projects/DIVA/boards/1?selectedIssue=DIVA-17)

## Description

I have completed the implementation of the Cart UI in the Flutter app. This PR includes the user interface for displaying items in the cart, as well as the layout for adjusting quantities and removing items.

## How to Test

1. Open the Flutter app.
2. Navigate to the Cart screen from the bottom navigation bar or after adding a product.
3. Verify that the cart UI displays a list of products with:
   * Product name and image.
   * Quantity selector (increment/decrement).
   * Remove button.
   * Total price calculation.
4. Check that the "Proceed to Checkout" button is visible.
5. Test the UI on different screen sizes to ensure responsiveness.

## Screenshots
![flutter_01](https://github.com/user-attachments/assets/c036874f-81b1-4b43-a6e9-335d23fafba7)


[DIVA-17]: https://nz-dev.atlassian.net/browse/DIVA-17?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
